logic/gateway.go

```diff
@@ -14,6 +14,14 @@ import (
 
 // CreateEgressGateway - creates an egress gateway
 func CreateEgressGateway(gateway models.EgressGatewayRequest) (models.Node, error) {
+	for i, cidr := range gateway.Ranges {
+		normalized, err := NormalizeCIDR(cidr)
+		if err != nil {
+			return models.Node{}, nil
```

Comment:
why not return the err (since it's not nil)?  If needs more context (as the err comes from a support function), could have a preamble (eg "error while normalizing CIDR, error was " + err...).